ceremonies/src/benchmarking.rs

```diff
@@ -0,0 +1,377 @@
+use crate::*;
+use encointer_primitives::communities::{
+	CommunityIdentifier, CommunityMetadata as CommunityMetadataType, Degree, Location, LossyInto,
+};
+use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
+use frame_support::{
+	assert_ok,
+	traits::{OnFinalize, OnInitialize},
+};
+use frame_system::RawOrigin;
+use sp_core::{sr25519, Pair};
+use sp_runtime::traits::UniqueSaturatedInto;
+
+pub const GENESIS_TIME: u64 = 1_585_058_843_000;
+pub const ONE_DAY: u64 = 86_400_000;
+pub const BLOCKTIME: u64 = 3_600_000;
+
+fn create_community<T: Config>() -> CommunityIdentifier {
+	let alice: T::AccountId = account("alice", 1, 1);
+	let bob: T::AccountId = account("bob", 2, 2);
+	let charlie: T::AccountId = account("charlie", 3, 3);
+
+	let location = Location { lat: Degree::from_num(1i32), lon: Degree::from_num(1i32) };
+
+	let bs = vec![alice.clone(), bob.clone(), charlie.clone()];
+	let community_meta: CommunityMetadataType = CommunityMetadataType {
+		name: "Default".into(),
+		symbol: "DEF".into(),
+		..Default::default()
+	};
+	encointer_communities::Pallet::<T>::new_community(
+		RawOrigin::Root.into(),
+		location,
+		bs.clone(),
+		community_meta.clone(),
```

Comment:
I think creating the metadata is unnecessary, you should just be able to create it in place. (You would not need to clone anyhow, as you don't use the metadata afterwards;

```suggestion
		CommunityMetadata::Default(),
```